src/moveit_visual_tools.cpp

```diff
@@ -416,57 +422,57 @@ bool MoveItVisualTools::publishEEMarkers(const geometry_msgs::Pose& pose, const
   // Does not require trigger() because publishing array auto-triggers
   if (!publishMarkers(ee_markers_map_[ee_jmg]))
   {
-    ROS_WARN_STREAM_NAMED(LOGNAME, "Unable to publish EE markers");
+    RCLCPP_WARN_STREAM(LOGGER, "Unable to publish EE markers");
     return false;
   }
 
   return true;
 }
 
-bool MoveItVisualTools::publishGrasps(const std::vector<moveit_msgs::Grasp>& possible_grasps,
+bool MoveItVisualTools::publishGrasps(const std::vector<moveit_msgs::msg::Grasp>& possible_grasps,
                                       const moveit::core::JointModelGroup* ee_jmg, double animate_speed)
 {
-  ROS_DEBUG_STREAM_NAMED(LOGNAME, "Visualizing " << possible_grasps.size() << " grasps with EE joint model group "
+  RCLCPP_DEBUG_STREAM(LOGGER, "Visualizing " << possible_grasps.size() << " grasps with EE joint model group "
                                                  << ee_jmg->getName());
 
   // Loop through all grasps
   for (std::size_t i = 0; i < possible_grasps.size(); ++i)
   {
-    if (!ros::ok())  // Check that ROS is still ok and that user isn't trying to quit
+    if (!rclcpp::ok())  // Check that ROS is still ok and that user isn't trying to quit
       break;
 
     publishEEMarkers(possible_grasps[i].grasp_pose.pose, ee_jmg);
 
-    ros::Duration(animate_speed).sleep();
+    rclcpp::sleep_for(std::chrono::milliseconds(int(animate_speed * 1000)));
```

Comment:
Maybe a better readable alternative:
```suggestion
    rclcpp::sleep_for(rclcpp::Duration::from_seconds(animate_speed));
```
Also, please extend the documentation that `animate_speed` is expecting seconds.
If we want to be really fancy, `animate_speed` should be a chrono type, but that is strictly out of scope of this PR. I'm thinking that this would be a good cleanup for the future, though.

Reply:
This also does not compile

Reply:
Add a small statement in doxy docs on animate speed should be given in seconds.